docs/CONTRIBUTING.md

```
@@ -1,37 +1,94 @@
+- Use **Black** formatter (`black .`) before pushing.
+- Validate HTML and CSS if you modify templates.
+- Keep commit messages clear and meaningful.
+
+Example commit:
+
+```bash
+git commit -m "Fix: Corrected login validation error"
+```
+
+### 6. Push Your Changes
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)